controllers/pachyderm_controller.go

```diff
@@ -338,6 +365,10 @@ func (r *PachydermReconciler) cleanupPachydermResources(ctx context.Context, pd
 	}
 
 	// delete cluster resources
```

Comment:
Does `validatePachyderm` delete cluster resources? Or is the comment out of date? Or does it refer to something farther down? (if so would it make sense to move the comment to the next stanza?)